db/structure.sql

```diff
@@ -2,12 +2,16 @@
 -- PostgreSQL database dump
 --
 
+-- Dumped from database version 9.5.3
+-- Dumped by pg_dump version 9.5.3
+
 SET statement_timeout = 0;
 SET lock_timeout = 0;
 SET client_encoding = 'UTF8';
 SET standard_conforming_strings = on;
 SET check_function_bodies = false;
 SET client_min_messages = warning;
+SET row_security = off;
```

Comment:
This was added in postgres 9.5 and will error on 9.4. You could just remove this, but the ideal thing would be to set pg_dump to dump for 9.4, but I'm not certain that's possible. You may have to try and install postgres version 9.4